cypress/e2e/saucedemo/Checkout.cy.ts

```diff
@@ -0,0 +1,26 @@
+/// <reference types="cypress" />
+
+import { STANDART_USER, PASSWORD } from "../../Constants";
+
+
+describe('checkout test', () => {
+  beforeEach(() => {
+    cy.visit('https://www.saucedemo.com/');
+    cy.get('[data-test="username"]').type(STANDART_USER)
+    cy.get('[data-test="password"]').type(PASSWORD)
+    cy.get('#login-button').click()
+
+  })
+  it('Checkout test', () => {
+    // cy.type('Cypress.io{enter}');
+    cy.get('#add-to-cart-sauce-labs-bike-light').click();
+    cy.get('.shopping_cart_badge').click();
+    cy.get('#checkout').click();
+    cy.get('[data-test="firstName"]').type('Anton'); //https://docs.cypress.io/api/cypress-api/custom-commands
+    cy.get('[data-test="lastName"]').type('Yu');
+    cy.get('[data-test="postalCode"]').type('220');
+    cy.get('#continue').click();
+    cy.get('#finish').click();
+  })
+
```

Comment:
Add final assert

Reply:
you can also add check on which page you stay https://docs.cypress.io/api/commands/title

Reply:
Or as in "second test" 
    cy.contains('Thank you for your order!');

Reply:
fixed